In [16]:
import pandas as pd

In [7]:
fruits = pd.DataFrame([[30,21]], columns =['Apples','Bananas'])
fruits

Apples  Bananas
0      30       21

In [8]:
fruit_sales = pd.DataFrame([[35,41],[21,34]], columns =['Apples','Bananas'], index =['2017 Sales','2018 Sales'])
fruit_sales

Apples  Bananas
2017 Sales      35       41
2018 Sales      21       34

In [22]:
ingredients = pd.Series(['4 cups','1 cup','2 large', '1 can'],index=[' Flour','Milk', 'Eggs','spam' ],name ='Dinner')
ingredients

 Flour     4 cups
Milk        1 cup
Eggs      2 large
spam        1 can
Name: Dinner, dtype: object

In [1]:
wine1 = pd.read_csv('Wine_review')
wine1.head(20)

NameError: name 'pd' is not defined

In [13]:
wine1.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 150930 entries, 0 to 150929
Data columns (total 11 columns):
 #   Column       Non-Null Count   Dtype  
---  ------       --------------   -----  
 0   Unnamed: 0   150930 non-null  int64  
 1   country      150925 non-null  object 
 2   description  150930 non-null  object 
 3   designation  105195 non-null  object 
 4   points       150930 non-null  int64  
 5   price        137235 non-null  float64
 6   province     150925 non-null  object 
 7   region_1     125870 non-null  object 
 8   region_2     60953 non-null   object 
 9   variety      150930 non-null  object 
 10  winery       150930 non-null  object 
dtypes: float64(1), int64(2), object(8)
memory usage: 12.7+ MB


In [14]:
wine1.describe()

Unnamed: 0         points          price
count  150930.000000  150930.000000  137235.000000
mean    75464.500000      87.888418      33.131482
std     43569.882402       3.222392      36.322536
min         0.000000      80.000000       4.000000
25%     37732.250000      86.000000      16.000000
50%     75464.500000      88.000000      24.000000
75%    113196.750000      90.000000      40.000000
max    150929.000000     100.000000    2300.000000

The data set used is the wine Review csv file. From the general information on the data set, it has a total of 11 columns and 150930 entries(rows). It is about the various types of wines and where the originate from, including the provinces and the varieties that are available.It also shows that from the varieties available their prices ranges from 4-2300. There is also a little description of the wine for those that will like to know more about the wines.

In [19]:
animals = pd.DataFrame({'Cows': [12, 20], 'Goats': [22, 19]}, index=['Year 1', 'Year 2'])
animals
animals.to_csv = ('animals.csv')